samples/Makefile.iopglobal_sample

```diff
@@ -110,9 +119,17 @@ $(IOP_OBJS_DIR)exports.o: $(IOP_OBJS_DIR)build-exports.c
 	$(DIR_GUARD)
 	$(IOP_C_COMPILE) $(IOP_IETABLE_CFLAGS) -c $< -o $@
 
-$(IOP_BIN): $(IOP_OBJS)
+$(IOP_BIN_ELF): $(IOP_OBJS)
+	$(DIR_GUARD)
+	$(IOP_C_COMPILE) -T$(IOP_LINKFILE) $(IOP_OPTFLAGS) -o $@ $(IOP_OBJS) $(IOP_LDFLAGS) $(IOP_LIBS)
+
+$(IOP_BIN_STRIPPED_ELF): $(IOP_BIN_ELF)
+	$(DIR_GUARD)
+	$(IOP_STRIP) --strip-debug --remove-section=.pdr -o $@ $<
```

Comment:
I think it should be `--strip-unneeded` instead

Reply:
Applied